### ISSUE TITLE
SmartCity-Accessibility Server Side Unit Tests #593 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#350]
Build 'SmartCity-Accessibility Server Side Unit Tests' is failing!

Last 50 lines of build :
```
[...truncated 11.82 KB...]
Running smartcity.accessibility.search.NearbyPlacesTest
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.055 sec <<< FAILURE! - in smartcity.accessibility.search.NearbyPlacesTest
nearByPlacesTest(smartcity.accessibility.search.NearbyPlacesTest)  Time elapsed: 0.051 sec  <<< ERROR!
java.lang.NullPointerException
	at smartcity.accessibility.search.NearbyPlacesTest.nearByPlacesTest(NearbyPlacesTest.java:32)

Running smartcity.accessibility.search.SearchTest
Tests run: 3, Failures: 0, Errors: 2, Skipped: 1, Time elapsed: 0.017 sec <<< FAILURE! - in smartcity.accessibility.search.SearchTest
test2(smartcity.accessibility.search.SearchTest)  Time elapsed: 0.005 sec  <<< ERROR!
java.lang.NullPointerException
	at smartcity.accessibility.search.SearchTest.test2(SearchTest.java:57)

test3(smartcity.accessibility.search.SearchTest)  Time elapsed: 0.007 sec  <<< ERROR!
java.lang.NullPointerException
	at smartcity.accessibility.search.SearchTest.test3(SearchTest.java:75)


Results :

Tests in error: 
  NearbyPlacesTest.nearByPlacesTest:32 » NullPointer
  SearchTest.test2:57 » NullPointer
  SearchTest.test3:75 » NullPointer
  LoginServiceTest.UserInfoSuccess » IllegalState Failed to load ApplicationCont...
  LoginServiceTest.loginSuccess » IllegalState Failed to load ApplicationContext
  LoginServiceTest.loginUserDoesNotExist » IllegalState Failed to load Applicati...
  LoginServiceTest.signupSuccess » IllegalState Failed to load ApplicationContex...
  LoginServiceTest.signupUserExist » IllegalState Failed to load ApplicationCont...

Tests run: 65, Failures: 0, Errors: 8, Skipped: 2

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 10:42 min
[INFO] Finished at: 2017-05-17T09:21:13+00:00
[INFO] Final Memory: 31M/209M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/sharedspace/SmartCity-Accessibility@2/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [KoralChapnik] a4958c33c5f1e98867951915a29db49515aebebe - added background to arthur page

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Unit%20Tests/593/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
